Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -472,6 +472,47 @@ private class PopupBlockingSettings: Setting {
     }
 }
 
+class ThemeImageSetting: Setting {
+    let profile: Profile
+    let prefKey = "themeImage"
+    var picker: UIImagePickerController? = nil
+
+    init(settings: SettingsTableViewController) {
+        self.profile = settings.profile
+        let title = NSLocalizedString("Theme image", comment: "Theme image setting")
+        super.init(title: NSAttributedString(string: title))
+    }
+
+    override func onConfigureCell(cell: UITableViewCell) {
```

Comment:
Nit: Don't need to override this method if nothing is happening in onConfigureCell